src/market_mad/remotes/linuxcontainers/monitor

```diff
@@ -190,6 +196,35 @@ class LinuxContainersMarket
 
         str << "#{name}=\"#{val}\"\n"
     end
+
+    def vm_tmpl64
+        template = JSON.generate(DEFAULTS[:template])
+        template = JSON.parse(template, :symbolize_names => true)
+        template = template_to_str(template)
+        Base64.strict_encode64(template)
+    end
+
+    def template_to_str(thash)
```

Comment:
Remove. Keep the template in String form in the default section. No need to mange this two formats